### ISSUE TITLE
subpage no longer linked in markdown table

### ISSUE THREAD

**Issue by higaski** [doxygen/doxygen#8155]
Using version 1.8.20 a subpage inside a markdown table does not produce a link when generating HTML output.

The issue can easily be reproduced with the following snippet:
```
/// \mainpage TestMain
/// This is the main page
///
/// \page page_contents Contents
/// This is the contents page
///
/// | Page               | Content   |
/// | ------------------ | --------- |
/// | \subpage page_test | Something |
///
/// \page page_test Test
/// This is the test page
///
```

The doxyfile used to create the HTML output:
[Doxyfile.txt](https://github.com/doxygen/doxygen/files/5482883/Doxyfile.txt)

I'm not sure with which version this stopped working though...

**Comment by albert-github** [comment]
With the version 1.8.20 I get the warning:
```
warning: unable to resolve reference to 'page_test\ilinebr' for \ref command
```
in the current master (67132b86a0d24cafab0fa400b1433f908efd4113) this problem is gone.
There have been a number of problems in respect to commands inside table cells (see also #7996).

This problem has been fixed but not released.

As a workaround you could try to use:
```
\subpage page_test ""
```
instead of:
```
\subpage page_test
```

**Comment by higaski** [comment]
Workaround works, thank you!

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
